### DISCUSSION TITLE
Card for Automations and Info

### DISCUSSION THREAD

**Discussion by pdhruv93** [shannonhochkins/ha-component-kit#35] [Q&A] [answered]
- Similar to the scenes card, can we have a card for running the automation?
- How to show data/info from the sensors for example Temperature, humidity?
- Also noticed one thing, when the entity is unavailable, can we disable the card, right now it remains in an active state.

**Comment by shannonhochkins** [comment]
Hi @pdhruv93!

I actually have a feature in the works for displaying multiple entites (similar to the EntitiesCard) which i'll most likely release tomorrow (screenshot below)

Great idea, I always thought the SceneCard might have been too generic, I will rename this to TriggerCard and have it support any entity, I'll release this with the changes for EntitiesCard :) 

As for the card showing active state, nice catch! I'll also fix this with the next release, hang tight please!

![Screenshot 2023-09-05 at 9 27 52 pm](https://github.com/shannonhochkins/ha-component-kit/assets/5105002/c41dbc9d-1d59-472b-8cb1-62b88a0911ba)

**Reply by pdhruv93** [reply]
Great! Let me know I will test

**Comment by shannonhochkins** [comment]
Just published a new version for core and components, let me know if this solves the issues you were having :)

**Reply by pdhruv93** [reply]
@shannonhochkins  Yes everything works perfect. Thanks for the quick release!
Just a few improvement if I would suggest:

- Can we get an icon parameter for TiggerCard component.
- Can we  parameterize the bounce time for the Tigger action switch? I think right now it bounces back to off state after 1minute. Or maybe use some other component other than power because right now it looks more like a on/off switch whereas it should only be a progress indicator.

**Reply by shannonhochkins** [reply]
@pdhruv93 

- icon is an easy one! Agreed, this card isn't as configurable as some of the others
- You're assumption is correct, currently it is activated based on the last activated time, which can take up to 60 seconds to swap it back to it's "off" state, i will also provide another icon for the toggle button too, I will have a think about how we can make this more configurable - leave it with me, should have something together in an hour or so

**Reply by shannonhochkins** [reply, marked as answer]
Alrighty - released more changes based on your suggestions, you can now provide an activeStateDuration prop if the default of 5000ms is not what you're after! Basically, after 5s it will reset back to default state, i've also included a wide range of configurable properties to customize the card futher
